0006/README.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 0006
+title: Resiliency in Tinkerbell Provisioner.
+status: Ideation
+authors: Chitrabasu Khare <chitrabasukhare89@gmail.com>
+---
+
+## Summary:
+
+This RFD proposes to have a resilient framework for Tinkerbell Provisioner.
+
+
+## Goals:
+
+- Bring resiliency in Tinkerbell. 
+- Tinkerbell as a setup should be highly available.
+
+## Content: 
+
+- We would like to have a framework where Tinkerbell as a solution is fault-tolerant.
+- All most all of the core services are managed by provisioner. So, it will be good to have a framework where provisioner is highly available.
+- It can be a multinode provisioner solution with active-active or active-passive implementation.
+- The Provisioner services should be partition tolerant during network failures or communication failure between provisioner nodes.
+- Additional logic would be required to sync services running inside provisioner nodes.
```

Comment:
Which one? can you more detailed?

Reply:
It is sync of tink-db service, i.e how will db service will sync volumes they are mounted and keep the cluster consistent.